komodo/release_transpiler.py

```diff
@@ -204,20 +204,31 @@ def main():
     matrix_parser.add_argument(
         "--release-folder",
         required=True,
+        type=dir_path,
         help="Folder with existing release file (default: None)",
     )
     matrix_parser.add_argument(
         "--override-mapping",
         required=True,
+        type=check_if_valid_file,
         help="File containing explicit matrix packages (default: None)",
     )
+
+    def checkValidPythonVersion(python_versions: str) -> list:
+        output_list = []
+        for item in python_versions.split(","):
+            if re.match(r"^[2,3](\.\d+)?$", item) is None:
+                raise TypeError(item)
+            output_list.append(item)
+        return output_list
+
     matrix_parser.add_argument(
         "--py_coords",
         help="""Comma delimitated list of python versions to be combined,
```

Comment:
A typo that might be fixed in this PR: delimitated -> delimited

Reply:
Good catch!